similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultPatientSimilarityView.java

```diff
@@ -368,7 +369,7 @@ public double getScore()
      *
      * @return the genotype similarity view for this pair of patients
      */
-    private GenotypeSimilarityView getGenotypeSimilarity()
+    private PatientGenotypeSimilarityView getGenotypeSimilarity()
     {
         if (this.matchedGenes == null) {
             this.matchedGenes = new RestrictedGenotypeSimilarityView(this.match, this.reference, this.access);
```

Comment:
I wonder why does the `Default` patient view return a `Restricted` genotype view...